src/test/java/picard/sam/CramCompatibilityTest.java

```diff
@@ -267,7 +274,7 @@ static void assertCRAM(final File outputFile) {
         try (InputStream in = new FileInputStream(outputFile)) {
             Assert.assertTrue(SamStreams.isCRAMFile(new BufferedInputStream(in)), "File " + outputFile.getAbsolutePath() + " is not a CRAM.");
         } catch (IOException e) {
-            e.printStackTrace();
+            throw new RuntimeIOException(e);
```

Comment:
I also changed this from a silent success to an exception since it seems like it could easily hide errors.